### ISSUE TITLE
Re-training yolov5 relu models

### ISSUE THREAD

**Issue by emilmgeorge** [Microchip-Vectorblox/VectorBlox-SDK#12]
Hi,

How can I retrain the yolov5 pre-trained models and run them using VectorBlox?

I tried the following:
- Tried using the ONNX model file provided in the VectorBlox tutorials with the [ultralytics/yolov5](https://github.com/ultralytics/yolov5) train.py script but it [requires](https://github.com/ultralytics/yolov5/blob/589edc7b012d45a5c8ad6231d7716f88cb6e43ca/train.py#L118) a pytorch(.pt) model file for training.
- Tried converting the ultralytics/yolov5 [pretrained pytorch models](https://github.com/ultralytics/yolov5#pretrained-checkpoints) to ONNX using ultralytics' export.py script and then using it with the VectorBlox sample script. But the ultralytics model uses SiLU activation. After conversion to ONNX, SiLU becomes Sigmoid + non-const Multiply which, if I understand correctly, is not supported with the Vectorblox SDK (please correct me if I'm wrong).

I'm using the following versions:

- ultralytics/yolov5: e40662ffdd80a6f108a62cf0d53d06085d943223
- VectorBlox-SDK: 024dc7cf6ca90b20d1fdad92ab57db5df86ffc5d (release-v1.4.2)

**Comment by Frostjord** [comment]
Hi,
Apparently the official YOLOv5 with SiLU was supposed to be supported with VectoBlox 1.4, but it seems that it's still not the case. 
However, you can change the activation functions in the common.py file, and retrain your model with ReLU.

```
class Conv(nn.Module):
    # Standard convolution
    def __init__(self, c1, c2, k=1, s=1, p=None, g=1, act=True):  # ch_in, ch_out, kernel, stride, padding, groups
    super().__init__()
    self.conv = nn.Conv2d(c1, c2, k, s, autopad(k, p), groups=g, bias=False)
    self.bn = nn.BatchNorm2d(c2)

    # Activation functions
    # self.act = nn.Tanh() if act is True else (act if isinstance(act, nn.Module) else nn.Identity())
    # self.act = nn.Sigmoid() if act is True else (act if isinstance(act, nn.Module) else nn.Identity())
    # self.act = nn.ReLU() if act is True else (act if isinstance(act, nn.Module) else nn.Identity())
    # self.act = nn.LeakyReLU(0.1) if act is True else (act if isinstance(act, nn.Module) else nn.Identity())
    # self.act = nn.Hardswish() if act is True else (act if isinstance(act, nn.Module) else nn.Identity())
    self.act = nn.SiLU() if act is True else (act if isinstance(act, nn.Module) else nn.Identity())
    # self.act = Mish() if act is True else (act if isinstance(act, nn.Module) else nn.Identity())
    # self.act = FReLU(c2) if act is True else (act if isinstance(act, nn.Module) else nn.Identity())
    # self.act = AconC(c2) if act is True else (act if isinstance(act, nn.Module) else nn.Identity())
    # self.act = MetaAconC(c2) if act is True else (act if isinstance(act, nn.Module) else nn.Identity())
```
